packages/render-fidelity-tools/src/components/rendering-scenario.ts

```diff
@@ -39,27 +41,66 @@ export class RenderingScenario extends LitElement {
     return `./results/${this.name}`;
   }
 
+  updated(changedProperties: Map<any, any>) {
+    super.updated(changedProperties);
+
+    if (this.analysis == null) {
+      this.loadAnalysis();
+    }
+  }
+
+  private toDecibels(rmsDistanceRatio: number) {
+    return (10 * Math.log10(rmsDistanceRatio)).toFixed(2)
+  }
+
+  private async loadAnalysis() {
+    const analysisPath = `${this.basePath}/analysis.json`;
+    this.analysis = await (await fetch(analysisPath)).json();
+  }
+
   render() {
     const {basePath} = this;
     const {width} = this.dimensions;
 
-    const images = [{
-                     name: 'model-viewer',
-                     description: '<model-viewer> (this version)',
-                     file: 'model-viewer.png'
-                   }].concat(this.goldens)
-                       .filter(golden => !this.exclude.includes(golden.name))
-                       .map(golden => html`
-<div class="screenshot">
-  <header>
-    <h2>${golden.description}</h2>
-  </header>
-  <div class="check"></div>
-  <img data-id="${this.name} ${golden.name}"
-       style="width:${width}px" src="${basePath}/${golden.file}">
-</div>`);
+    const images =
+        [{
+          name: 'model-viewer',
+          description: '<model-viewer> (this version)',
+          file: 'model-viewer.png'
+        }].concat(this.goldens)
+            .filter(golden => !this.exclude.includes(golden.name))
+            .map(
+                (golden, index) => html`
```

Comment:
I wonder if this is starting to reach critical mass in terms of the amount of embedded HTML.  For Filament's material debugger, I used `<template>` in some situations:

https://github.com/google/filament/blob/main/libs/matdbg/web/index.html#L27L39